Rnwood.Smtp4dev/appsettings.json

```diff
@@ -111,7 +112,14 @@
     // - Rejects messages that include in the subject
     // message.subject.includes("19") ? 441 : null
     // - Rejects messages that include 19 with a 441, otherwise accepts
-    "MessageValidationExpression": ""
+    "MessageValidationExpression": "",
+
+    "Users": [
+      //{
+      //  "Username": "username",
+      //  "Password": "password"
+      //}
+    ]
```

Comment:
I am wondering how would this option be translated into `docker-compose.yml` file config in the environment variables?

I tried the following but it's not working:
```
  smtp4dev:
    image: rnwood/smtp4dev:3.3.6
    environment:
      - ServerOptions__Port=587
      - RelayOptions__SmtpPort=587
      - ServerOptions__Users="[{"Username":"email@host.com","Password":"YouWillNeverKnow"}]"
    ports:
      - 5001:80
```

Reply:
update: as a workaround for now I've used the user mounted setting in docker and set my desired settings first hand in json:
```json
  smtp4dev:
    image: rnwood/smtp4dev:3.3.6
    ports:
      - 5001:80
    volumes:
      - ./smtp4dev.json:/smtp4dev/appsettings.json
```